infra/aws/terraform/prow-build-cluster/README.md

```diff
@@ -243,3 +243,31 @@ make destroy
 
 If you want to remove roles used for EKS creation go to `../iam/<aws_account_name>` and run `terraform destroy` command there.
 
+
+## GitOps
+
+To sync state from git repo into EKS cluster we use [FluxCD](https://fluxcd.io/).
+
+Flux Kustomizations and HelmReleases for manifests are stored inside the `./resources` directory and have been generated with use of `./hack/flux-update.bash`. On top of that, the script prepares manifests for the GitOps Tool Kit and can be used to update its version.
+
+The `flux-system` namespace contains all GitOps Tool Kit componenets as well as all Flux Kustomizations.
+
+Flux Helm Releases ought to be deployed in the same namespaces as manifests they create. As a convention, all Flux Helm Releases have to be prefixed with `flux-hr-`. This convention is leveraged by automation.
```

Comment:
Is this convention coming from Flux or us? It might be useful to clarify that in the document.